cmd/main.go

```diff
@@ -201,6 +202,26 @@ func maxDuration(dur time.Duration, durations ...time.Duration) time.Duration {
 	return ret
 }
 
+func WrapServeError() func(rw http.ResponseWriter, r *http.Request, err error) {
+	compositeErrorDefaultMessage := "validation failure list:"
```

Comment:
I wouldn't want to depend on a string defined in another project

Reply:
changed